build/templates/session.py.mako

```diff
@@ -100,7 +100,7 @@ class AttributeEnum(object):
 
     def __set__(self, obj, value):
         if type(value) is not self.attribute_type:
-            raise TypeError('Value mode must be of type ' + str(self.attribute_type))
+            raise TypeError('Must be ${module_name}.' + str(self.attribute_type.__name__) + ' not ' + str(type(value).__name__))
```

Comment:
>>> print("a" + f)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: must be str, not Frank

So I would change Must to must